### DISCUSSION TITLE
Automatically assigned DOI not working

### DISCUSSION THREAD

**Discussion by ALedl** [knowledgefutures/pubpub#2079] [Support] [answered]
I created this issue with 9 pubs: https://0277.pubpub.org/wisskom2022-wie-macht-ihr-das--offentliche-und-wissenschaftliche-bibliotheken-im-dialog. All pubs have been deposited to Crossref, but for one, the DOI just won't work. I deleted the pub and recreated and registered it again. The result was the same. I just can't figure out where the difference is from the other pubs.

This is the pub: https://0277.pubpub.org/pub/wm9dq1e7/release/1

And this is the DOI: https://doi.org/10.21428/1bfadeb6.4112166b

**Comment by gabestein** [comment, marked as answer]
Hello! Thanks for writing in, and sorry about this issue. I took a look at the logs, and crossref is complaining that the ORCID ID for the author is invalid because it has an errant space at the end of it. That's something we can ultimately fix by stripping whitespace, but for now, I went ahead and fixed it for the user so you should be set to redeposit.

**Reply by ALedl** [reply]
Thank you for your quick help. That has solved the problem.
